tools/ci_build/github/android/build_aar_and_copy_artifacts.sh

```diff
@@ -12,23 +12,32 @@ ls /build
 ls /build/deps
 # build the AAR package, using the build settings under /home/onnxruntimedev/.build_settings/
 # if there is also include_ops_and_types.config exists in the same folder, use it to build with included ops/types
-if [ -f "/home/onnxruntimedev/.build_settings/include_ops_and_types.config" ]; then
-    python3 /onnxruntime_src/tools/ci_build/github/android/build_aar_package.py \
-        --build_dir /build \
-        --config $BUILD_CONFIG \
-        --android_sdk_path /android_home \
-        --android_ndk_path /ndk_home \
-        --include_ops_by_config /home/onnxruntimedev/.build_settings/include_ops_and_types.config \
-        /home/onnxruntimedev/.build_settings/build_settings.json
-else
-    python3 /onnxruntime_src/tools/ci_build/github/android/build_aar_package.py \
-        --build_dir /build \
-        --config $BUILD_CONFIG \
-        --android_sdk_path /android_home \
-        --android_ndk_path /ndk_home \
-        /home/onnxruntimedev/.build_settings/build_settings.json
+
+BUILD_SCRIPT="/onnxruntime_src/tools/ci_build/github/android/build_aar_package.py"
+BUILD_SETTINGS="/home/onnxruntimedev/.build_settings/build_settings.json"
+INCLUDE_OPS_CONFIG="/home/onnxruntimedev/.build_settings/include_ops_and_types.config"
+
+ANDROID_SDK_HOME="/android_home"
+ANDROID_NDK_HOME="/ndk_home"
+QNN_HOME="/qnn_home"
```

Comment:
nit: if we run this on a CI machine image that has '/qnn_home' setup (to avoid downloading each time) is there potential for QNN to be unintentionally included in the package? may be safer to require the QNN inclusion to be manually enabled (e.g. via an arg to the script, or the default value of QNN_HOME could be an invalid path).

Reply:
the path `/qnn_home` is within the docker container. whether it exists should be controlled by whether it gets mounted when calling `docker run`.